wiki/History_of_osu!/2012/id.md

```diff
@@ -87,21 +85,21 @@ Tautan terkait:
 
 Pada 2012-12-21, osu! diberikan pembaruan yang cukup besar. Perubahan yang paling menonjol adalah sebagai berikut:
 
-1.  Dukungan widescreen
-2.  Layar pemuatan di awal beatmap
-3.  Layar pemilihan lagu yang lebih lancar
-4.  Pengaturan Fun Spoiler diubah menjadi [Pengaturan Visual](/wiki/Visual_Settings) dan ditempatkan di bagian bawah layar
-5.  Tidak ada mod Video yang dipindahkan ke Pengaturan Visual
+1. Dukungan widescreen
+2. Layar pemuatan di awal beatmap
+3. Layar pemilihan lagu yang lebih lancar
+4. Pengaturan Fun Spoiler diubah menjadi [Pengaturan Visual](/wiki/Visual_Settings) dan ditempatkan di bagian bawah layar
+5. Tidak ada mod Video yang dipindahkan ke Pengaturan Visual
 
-Masalah teknis dan bug telah diprediksi dari awal. Dukungan widescreen, meskipun berfungsi, tetapi menyebabkan beberapa gambar dan video terlihat aneh (kepala karakter tidak terlihat, hilang atau sebagian tersembunyikan oleh teks) bagian atas dan bawah terpotong. [Storyboard](/wiki/Storyboard) saat ini belum mendukung widescreen, sebagai gantinya, letterbox atau bilah hitam muncul di sisi yang menyebabkannya terlihat sempit. Hal ini menciptakan masalah untuk mode [Catch the Beat](/wiki/Catch_the_Beat) karena buah akan *melewati* bar hitam yang harusnya berada di perbatasan saat menggunakan widescreen. Beberapa storyboard juga tidak berfungsi dengan baik (beberapa gambar tidak menempati area bermain, tetapi sebaliknya, memperpanjang ke bar hitam). Masalah saat ini telah diketahui dan telah diperbaiki sebagai solusi atas masalah yang didiskusikan dan ditemukan.
```

Comment:
probably revert all the game mode name changes as some are using the old names on purpose